lumigator/infra/mzai/helm/lumigator/templates/mistral-secret.yaml

```diff
@@ -1,15 +1,20 @@
-{{- include "lumigator.mistral-default-secret" . -}}
-{{- if .Values.mistralAPIKey -}}
-{{- if .Values.existingMistralAPISecret -}}
-{{- fail "A Mistral key and an existing Mistral Secret name cannot be provided at the same time" -}}
+{{- include "lumigator.lumigator-default-secret" . -}}
+{{- if or (.Values.mistralAPIKey) (.Values.openaiAPIKey) -}}
+{{- if .Values.existingSecret -}}
+{{- fail "A key and an existing Secret name cannot be provided at the same time" -}}
 {{- end -}}
 apiVersion: v1
 kind: Secret
 metadata:
-  name: {{ include "lumigator.mistral-secret-name" . }}
+  name: {{ include "lumigator.lumigator-secret-name" . }}
   labels:
     {{- include "lumigator.labels" . | nindent 4 }}
 type: Opaque
 data:
+{{- if .Values.mistralAPIKey -}}
   {{ .Consts.mistralSecretKey }}: {{ .Values.mistralAPIKey | b64enc | quote }}
+{{- end -}}
+{{- if .Values.openaiAPIKey -}}
```

Comment:
add openaiAPIKey to reademe

Reply:
It's already there! https://github.com/mozilla-ai/lumigator/pull/246/files/e67314bbd13b1d1ffa3ada0320607fe43f24aa9a#diff-d2cd8afe06d80614f32f5ff5afcd1f2cbb81154ae506f51cbc38babe4a903718R40